### ISSUE TITLE
Changing default site ID

### ISSUE THREAD

**Issue by Lycurgue** [humanmade/network-media-library#53]
Hi,

I'd like to change the default site ID from 2 to 1.
From the installation instructions I have to use the following filter:

```
add_filter( 'network-media-library/site_id', function( $site_id ) {
    return 123;
} );
```

Can you tell me where I have to put this code in _network-media-library.php_?

I also saw in _network-media-library.php_ in line 56

```
 const SITE_ID = 2;
```

Do I need to change _const SITE_ID = 2_; 

Thanks,

François Maurice

**Comment by tmort** [comment]
I just installed the plugin and placed the filter inside my functions.php, changing the ID to "1" for my main site. Try adding it to your themes functions.php and see if that does the trick.

**Comment by Lycurgue** [comment]
Hi @tmort 

Finally, I simply modify const _SITE_ID = 2_ to const _SITE_ID = 1_ and it does the trick !

But, I'm curious. Where you put the filter in the functions.php file ?

Thanks,

François

**Comment by johnbillion** [comment]
@Lycurgue If you're not familiar with actions and filters in WordPress then I'd suggest reading a few articles about them to familiarise yourself. They're one of the most important features of WordPress.

You can place this code anywhere in the functions.php file of your theme, or you can create a mu-plugin and placed the code in there. There are lots of resources online about these processes.

Good luck!

**Comment by Lycurgue** [comment]
Thanks @johnbillion,

I'll do my homework ;-)

François

**Comment by regivanx** [comment]
Hello, forgive me for insisting, but I do not know PHP programming. I use Bash, wp-cli or Composer, but I do not know how to program a filter hook.

Would it not be possible to say: "write this in a file with such name, and put it in such folder"?

I converted my wordpress site into a network, added a second site, and then installed Network Media Library with Composer. Adding images to the site library id = 1 or id = 2 does not echo in the other library.

Both sites have different themes.

I tried to create a mu-plugin by writing a site_id.php file:

`<?php add_filter( 'network-media-library/site_id', function( $site_id ) { return 1; } ); ?> `

and I put it in wp-content/mu-plugins.

It does not work.

I do not know what to do now, I do not know PHP, I do not know where to look.
